GEOSmetforce_GridComp/LDAS_Forcing.F90

```diff
@@ -5539,6 +5635,8 @@ subroutine get_GEOS_forcing_filename(fname_full,file_exists, date_time, daily_fi
     character( 16) :: time_stamp
     character(  4) :: YYYY,  HHMM, day_dir
     character(  2) :: MM,    DD  
+    character(  8) :: S2S3_init_YYYYMMDD  ! S2S3 initial date, e.g. "20160101" 
+    character(  4) :: S2S3_ens_num        ! S2S3 ensemble member, e.g. "ens1" 
```

Comment:
@gmao-qliu : For S2S3, is there a chance that there are more than 10 ensemble members with the same start date?  I forget how the "ensemble" is constructed.  I think a given month's ensemble is made up of forecasts that start on different days, with each start date having at most a few (<10) members.  If that's the case, we should be ok.  Just double-checking

Reply:
@gmao-qliu:  For the case of 10 ens members on the last start date of the month, is the format of the file name substring:
1) "ens1", "ens2", ..., "ens9", "ens10"?  
2) "ens0", "ens1", ..., "ens8", "ens9"? 
3) "ens01", "ens02", ..., "ens09", "ens10"? 

I can only hope it's not case 1...

Reply:
correction: the special case is 15 members not 10, and the format is case 1, i.e. "ens1", "ens2", ..."ens10", "ens11', ..